frontends/concrete-python/examples/tfhers-cml/test.py

```diff
@@ -0,0 +1,191 @@
+import os
+from functools import partial
+
+import click
+import numpy as np
+
+from concrete import fhe
+from concrete.fhe import tfhers
+
+from numpy.random import randint
+
+# FIXME: should we move this to Concrete library directly, hidden to the user
+def get_tfhers_params_and_type_and_int(precision):
+    tfhers_params = tfhers.CryptoParams(
+        lwe_dimension=LWE_DIM,
+        glwe_dimension=GLWE_DIM,
+        polynomial_size=POLY_SIZE,
+        pbs_base_log=PBS_BASE_LOG,
+        pbs_level=PBS_LEVEL,
+        lwe_noise_distribution=LWE_NOISE_DISTR,
+        glwe_noise_distribution=GLWE_NOISE_DISTR,
+        encryption_key_choice=ENCRYPTION_KEY_CHOICE,
+    )
+    tfhers_type = tfhers.TFHERSIntegerType(
+        is_signed=False,
+        bit_width=precision,
+        carry_width=CARRY_WIDTH,
+        msg_width=MSG_WIDTH,
+        params=tfhers_params,
+    )
+    tfhers_int = partial(tfhers.TFHERSInteger, tfhers_type)
+
+    return tfhers_params, tfhers_type, tfhers_int
+
+# FIXME Params: users shouldn't change them, should we hide it
+LWE_DIM = 909
+GLWE_DIM = 1
+POLY_SIZE = 4096
+PBS_BASE_LOG = 15
+PBS_LEVEL = 2
+MSG_WIDTH = 2
+CARRY_WIDTH = 3
+ENCRYPTION_KEY_CHOICE = tfhers.EncryptionKeyChoice.BIG
+LWE_NOISE_DISTR = 0
+GLWE_NOISE_DISTR = 2.168404344971009e-19
+
+assert GLWE_DIM == 1, "glwe dim must be 1"
+
+# Options: the user can change the following
+# FIXME: explain FHEUINT_PRECISION, ie can it be changed
+FHEUINT_PRECISION = 8
+tfhers_params, tfhers_type, tfhers_int = get_tfhers_params_and_type_and_int(FHEUINT_PRECISION)
+
+# Describe the function you want to apply, on Concrete ciphertexts
+def server_side_function_in_concrete(concrete_x, concrete_y):
+    return (concrete_x + concrete_y) % 47
+
+# The user must specify the range of the TFHE-rs inputs
+# FIXME: why can't we set the limit at 256? It's needed for FHEUint8
```

Comment:
There is an issue here if I try to pick a random 8b byte (not 7b), I'll ask for help
CC @youben11

Reply:
what's the issue?

Reply:
I got this: https://github.com/zama-ai/concrete-internal/issues/877